src/components/contacts-list/contacts-list.container.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { fetchContacts } from '../../actions';
+import ContactsList from './contacts-list.component';
+
+const mapStateToProps = state => ({
+  contacts: Object.values(state.data.contacts)
```

Comment:
I think you shouldn't make anymore conversions on data from store. This means that whenever you want to access the contacts list from the store, you'll need to make this conversion. Instead, you could do this conversion when setting the data in the store (so you already have the contacts as an array on the store)